cogip/serialcontroller.py

```diff
@@ -164,25 +164,35 @@ def get_pose(self):
         self.serial_port.write(b"_pose\n")
 
         pose_found = False
-        while not self.exiting and not pose_found:
+        attempt = 0
+        while not self.exiting and not pose_found and attempt < 20:
             line = self.serial_port.readline().rstrip().decode(errors="ignore")
+            if line[0] == ">":
+                continue
             try:
                 pose = PoseCurrent.parse_raw(line)
                 self.signal_new_robot_position.emit(pose)
                 pose_found = True
             except ValidationError:
-                pass
+                attempt += 1
+                self.signal_new_console_text.emit(line)
+                # print(f"parse failed: {line}")
 
     def get_dyn_obstacles(self):
         self.serial_port.write(b"_dyn_obstacles\n")
 
         obstacles_found = False
-        while not self.exiting and not obstacles_found:
+        attempt = 0
+        while not self.exiting and not obstacles_found and attempt < 20:
```

Comment:
it would be good to at least set this treshold into a constant.

Reply:
Done